test/e2e/limit_range.go

```diff
@@ -61,7 +61,11 @@ var _ = Describe("LimitRange", func() {
 		Expect(err).NotTo(HaveOccurred())
 		for i := range pod.Spec.Containers {
 			err = equalResourceRequirement(expected, pod.Spec.Containers[i].Resources)
-			Expect(err).NotTo(HaveOccurred())
+			if err != nil {
+				// Print the pod to help in debugging.
+				Logf("Pod %+v does not have the expected requirements", pod)
```

Comment:
equalResourceRequirement() does not get the pod. It only gets the expected and actual requirements which it anyways prints in Logf().
I want to see if other fields in the pod are being set or not.